source/administration/telemetry.rst

```diff
@@ -56,15 +56,27 @@ Server Configuration Settings
 Commercial License Information (Enterprise Edition Only)
   Information about commercial license key purchased or trial license key used for Enterprise Edition servers: Company ID, license ID, license issue date, license start date, license expiry date, number of licensed users, license short name (E10 vs E20), list of unlocked Enterprise features.
 
+Channel Moderation Configuration Information (Enterprise Edition Only)
+  Information related to channel moderation, including number of channel schemes, number of channels with posting messages disabled for users or guests, number of channels with emoji reactions disabled for users or guests, number of channels with managing members disabled, number of channels with channel mentions disabled for users or guests.
+
+Groups Configuration Information (Enterprise Edition Only)
```

Comment:
@mkraft Please review groups telemetry for correctness (from https://github.com/mattermost/mattermost-server/pull/13736)

Reply:
@jasonblais LGTM 👍